commcare_connect/opportunity/visit_import.py

```diff
@@ -132,7 +132,7 @@ def _bulk_update_visit_status(opportunity: Opportunity, dataset: Dataset):
                 if visit.status != status:
                     visit.update_status(status)
                     visit.reason = reason
-                    if opportunity.managed and status in ["approved", "rejected"]:
```

Comment:
is the review stll created, there just isn't a date on the visit anymore?

Reply:
This was a change to make the rejected reason field work as the justification. I reverted these changes in [7a3bfd8](https://github.com/dimagi/commcare-connect/pull/383/commits/7a3bfd8224a27efb62e59f98028645d5a44db573)